tagged-localbuild.xml

```diff
@@ -33,7 +33,7 @@
   <project name="device-sony-pdx213" path="device/sony/pdx213" remote="sony" revision="47cb16d82a5cb8a91fcdacf313b1ecdd4ff9401d" upstream="r-mr1" dest-branch="r-mr1" groups="device"/>
   <project name="device-sony-pioneer" path="device/sony/pioneer" remote="sony" revision="f5e43dffef0f7ce6c8d1f7c5effc9ff8edfbf272" upstream="r-mr1" dest-branch="r-mr1" groups="device"/>
   <project name="device-sony-poplar" path="device/sony/poplar" remote="sony" revision="013d1bd25310dd9be08086ac03cac9714cd439cd" upstream="r-mr1" dest-branch="r-mr1" groups="device"/>
-  <project name="device-sony-seine" path="device/sony/seine" remote="sony" revision="ddee8d1a7793ebef236bf000447ff1ffc8802b5e" upstream="r-mr1" dest-branch="r-mr1" groups="device"/>
+  <project name="device-sony-seine" path="device/sony/seine" remote="sony" revision="7c53a5e365588de99e76f832172d37e9b4529270" upstream="r-mr1" dest-branch="r-mr1" groups="device"/>
```

Comment:
This is a result of a forced push upstream, net result is the same checkout of files.